Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -27,6 +28,10 @@ public class ListViewRenderer : ViewRenderer<ListView, AListView>, SwipeRefreshL
 		IListViewController Controller => Element;
 		ITemplatedItemsView<Cell> TemplatedItemsView => Element;
 
+		internal ListViewRenderer(IntPtr handle, JniHandleOwnership ownership) : base(handle, ownership)
```

Comment:
This ctor can be triggered by exiting the test and returning to the test. Elsewhere, XF allows objects to be resurrected using this ctor for the purpose of allowing dispose to be called. While I'm not exactly sure what this is all about, it seems innocuous; I followed the convention.